lib/sprite.rb

```diff
@@ -7,8 +7,6 @@ def self.character(filename, **opts)
     self[File.join('character', filename), **opts]
   end
 
-  private
```

Comment:
This is the only bit I'm not sure about, but the game seems to work fine without this. Rubocop called this "useless" unless it's moved inside a specific block type.